app/models/ability.rb

```diff
@@ -4,27 +4,28 @@ class Ability
   def initialize(user)
     if user
       can :manage, User, id: user.id
-      can :manage, GroupedIssue do |group|
-        user.is_owner_of?(group.website)
-      end
+      can :manage, GroupedIssue
       can :manage, Issue do |issue|
-        user.is_owner_of?(issue.group.website)
+        user.is_owner_of?(issue.website)
       end
       can :manage, Message do |message|
-        user.is_owner_of?(message.issue.group.website)
+        user.is_owner_of?(message.subscriber.website)
       end
```

Comment:
we need to allow users to do these:
- `:manage, Message`
- `[:read, :update], Issue`
- `[:read, :update], GroupedIssue`
We need to allow owners to:
- `:manage, Message`
- `:manage, GroupedIssue`
- `:manage, Issue`

We are talking about users and owners of the website that these records belong to.